src/report_generators/fake_header_report_generator.py

```diff
@@ -1,9 +1,5 @@
 from src.report_generators.base_report_generator import BaseReportGenerator
-from src.helpers.preprocess_text import extract_subtext
-
-import ast
-import pandas as pd
-from bs4 import BeautifulSoup
+from src.helpers.preprocess_text import extract_subtext, extract_html_tag
```

Comment:
General points (which I've mentioned elsewhere as well): this is a breaking change as the definition for this method hasn't been made yet. I'd swap the order of commits you've made so that the new method and tests are added first (and nothing else breaks), then you fix things here to use the new code.

Reply:
Thanks, Kelvin! I've swapped the commits now.

On the commit message being too long, I'm assuming you mean the body text? I don't think it actually is. The git commit message separates the commit header from the commit body text. The commit body text can be longer than 72 characters, the guidance you linked to recommends that it word-wraps at 72 characters. Is this what you mean instead?